### PR TITLE
Add blog post `Simple versioning with git tags`

### PR DESCRIPTION
**Pull request by grihabor**

### CODE REVIEW

File: blog/2024-04-27-simple-versioning-with-git-tags/index.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+multiple times, but all the methods look complicated or don't cover all the use
+cases. 
+
+- [Generating version tags from Git](../../docs/docs/using-pants/generating-version-tags-from-git.mdx)
+- [Custom `python_artifact()` kwargs](../../docs/docs/writing-plugins/common-plugin-tasks/custom-python-artifact-kwargs.mdx)
+- [vcs_version](../../reference/targets/vcs_version.mdx) target
+- [Using env vars to include dynamic data in tags](../../docs/docs/docker/tagging-docker-images.mdx#using-env-vars-to-include-dynamic-data-in-tags)
+
+To make it work we need one more pants feature [hidden in the
+docs](../../docs/docs/using-pants/key-concepts/options.mdx#pantsbootstrap-file)
```
Comment (grihabor) [inline blog/2024-04-27-simple-versioning-with-git-tags/index.mdx:19]: I'm not sure this link will work

**Comment by benjyw** [comment]
Thanks @grihabor , this is great! Do you mind if I tweak it a bit?

**Comment by grihabor** [comment]
> Do you mind if I tweak it a bit?

Go ahead, I don't mind :)

File: blog/2024-04-27-simple-versioning-with-git-tags/index.mdx [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
+    authors: [grihabor]
+    tags: []
+---
+
+# Simple versioning with git tags
+
+How do I set a docker image version based on git tag? Or python distribution
```
[the remark this answers is not in the document]
Comment (grihabor) [reply blog/2024-04-27-simple-versioning-with-git-tags/index.mdx:8]: It's a bit different style, but ok :)

[the remark this answers is not in the document]
Comment (grihabor) [reply blog/2024-04-27-simple-versioning-with-git-tags/index.mdx]: https://github.com/pantsbuild/pantsbuild.org/pull/197/commits/fab493bf7241a03c2133697fc1d22014c274271a

[the remark this answers is not in the document]
Comment (benjyw) [reply blog/2024-04-27-simple-versioning-with-git-tags/index.mdx]: > It's a bit different style, but ok :)

Yeah, just wanted to keep it a bit more positive :)

**Review by benjyw — APPROVED** [review]
Looks great, thanks!

**Comment by benjyw** [comment]
Not sure what the deal is with the CI failure, I will investigate.

**Comment by grihabor** [comment]
Yeah, looks good now

**Comment by benjyw** [comment]
> Yeah, looks good now

Probably just had to merge main into the branch.

**Comment by grihabor** [comment]
meh, links don't work https://www.pantsbuild.org/blog/2024/04/27/simple-versioning-with-git-tags

**Comment by grihabor** [comment]
here is the fix https://github.com/pantsbuild/pantsbuild.org/pull/202

**Pull request merged**